packages/ui/src/ui/store/reducers/global/cluster-ui-config.ts

```diff
@@ -7,4 +7,6 @@ export interface ClusterUiConfig {
     enable_maintenance_api_proxies?: boolean;
 }
 
-export const defaultClusterUiConfig: Partial<ClusterUiConfig> = {};
+export const defaultClusterUiConfig: Partial<ClusterUiConfig> = {
+    web_json_value_format: 'yql',
```

Comment:
@ma-efremoff I want to ask, is it normal to enable this setting for everyone?

Reply:
The option was required when not all versions of running clusters supported the format. It might be removed 
 if we sure all clusters are are ready for that.

Reply:
It looks like we can to remove the option.